litellm/main.py

```diff
@@ -40,7 +40,7 @@ def completion(
     # model specific optional params
     # used by text-bison only
     top_k=40, request_timeout=0, # unused var for old version of OpenAI API
-  ):
+  ) -> ModelResponse:
```

Comment:
Seems like type is not inferred here. Add explicit return type.

Reply:
for OpenAI we don't return `ModelResponse` , can you add a test in test_completion to ensure this doesn't break openai responses. 

essentially both of these should still work for OpenAI
```python
print(response.choices[0].message.content)
print(response['choices'][0]['message']['content'])
```